src/actions/dashboardData.js

```diff
@@ -0,0 +1,48 @@
+import {checkHttpStatus }from '../utils/apiUtils.js'
+import dashboardConstants from '../constants/dashboardConstants'
+import Axios from 'axios'
+import history from '../history';
+import { Redirect } from 'react-router-dom'
+
+export const dashboardData = {
+    getDashboardData
+    // logout,
+    // register,
+    // getAll,
+    // delete: _delete
+}
+
+export function getDashboardData() {
+    return dispatch => {
+        let token1  = localStorage.getItem('token');
```

Comment:
Give proper veriable names example: token1 should be "access_token".
Make it const not let.